tools/postCommit.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { resolve, extname } = require('path');
+const { spawn, exec } = require('child-process-promise');
+const { stripIndent } = require('common-tags');
+const root = resolve(__dirname, '../');
+const Listr = require('listr');
+const fs = require('mz/fs');
+const { existsSync } = require('fs');
+
+/**
+ * Checks if the files in question have associated diffs
+ */
+async function hasDiff(files) {
+  let { stdout } = await exec(`git diff ${files.join(' ')}`, { cwd: root });
```

Comment:
You can pass `--quiet` and then check the exit code: https://git-scm.com/docs/git-diff.

You can then likely skip passing in the filenames, which will prevent me from breaking this in the future by maliciously using filenames that contain single quotes.